### PR TITLE
added grid component

### PR DESCRIPTION
**Pull request by nikhiljugale007**
Added grid component:
50-50 grid
70-30 grid
30-70 grid
4 Item responsive grid

PREVIEW:: https://vercel-ui-preview.netlify.app/component-lib/grid.html

### CODE REVIEW

**Comment by rdthetha** [comment]
In the preview for the 70:30 and 30:70 grid , the section2 is overflowing from the container. Can u check?

File: grid.css [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+	grid-template-columns: repeat(12, 1fr);
+	gap: 2rem;
+}
+.grid-50-50 {
+	grid-template-columns: 1fr 1fr;
+}
+.grid-70-30 {
+	width: 100%;
+	display: grid;
+	grid-template-columns: 70% 30%;
```
Comment (Ankur9669) [inline grid.css]: You can remove
width: 100%;
display: grid;
From grid-70-30 as it is already present in grid class

**Review by Ankur9669 — CHANGES_REQUESTED** [review]
The code looks nice, not many major changes needed👍👍

**Comment by nikhiljugale007** [comment]
> In the preview for the 70:30 and 30:70 grid , the section2 is overflowing from the container. Can u check?

resolved .
Thanks for the suggestion.

File: grid.css (hunk 1 above)
Comment (nikhiljugale007) [reply grid.css]: Thanks. Removed.

**Pull request merged**